src/polish/StructuralPol.gf

```diff
@@ -57,6 +57,7 @@ lin
   in8front_Prep  = mkPrep "przed" Instr;
   in_Prep  =  mkPrep "w" Loc;
   it_Pron    = pronOno;
+  it_ReflPron    = pronRefl;
```

Comment:
There's no abstract syntax function for it_ReflPron, so this won't be of any use. It won't cause harm either, there will just be a message `Warning: function it_ReflPron is not in abstract` when compiling. You can just use the oper `pronRefl` directly in your application grammar, if you open PronounMorphoPol in it.